src/comparator/file_set_comparator.py

```diff
@@ -215,7 +215,7 @@ def _compare_resources(self, fs_original, fs_update):
                 category=FindingCategory.RESOURCE_DEFINITION_ADDITION,
                 proto_file_name=resources_update.types[resource_type].proto_file_name,
                 source_code_line=resources_update.types[resource_type].source_code_line,
-                message=f"A file-level resource definition `{resource_type}` has been added.",
+                message=f"A new resource definition `{resource_type}` has been added.",
```

Comment:
Since we currently put all resources defined in file-level and message-level, the error message should be updated.